SmartEVSE-3/src/evse.cpp

```diff
@@ -3251,11 +3306,14 @@ void StartwebServer(void) {
         doc["settings"]["starttime"] = (DelayedStartTime.epoch2 ? DelayedStartTime.epoch2 + EPOCH2_OFFSET : 0);
         doc["settings"]["stoptime"] = (DelayedStopTime.epoch2 ? DelayedStopTime.epoch2 + EPOCH2_OFFSET : 0);
         doc["settings"]["repeat"] = DelayedRepeat;
-        
-        doc["car_modem"]["ev_initial_soc"] = InitialSoC;
-        doc["car_modem"]["ev_full_soc"] = FullSoC;
-        doc["car_modem"]["ev_bulk_soc"] = BulkSoC;
-        doc["car_modem"]["computed_soc"] = ComputedSoC; 
+        doc["settings"]["required_evccid"] = RequiredEVCCID;
+        doc["ev_state"]["initial_soc"] = InitialSoC;
+        doc["ev_state"]["remaining_soc"] = RemainingSoC;
```

Comment:
RemainingSoC is actually the remaining % to charge before it reaches 100%. It looks easy enough to calculate on (for example) Home Assistant or other integrations, but I noticed when creating dashboards/automations it can be very useful to have such numbers already calculated for you without requiring any post-processing. But it can definitely be considered redundant.

Reply:
Your last changes made it a bit more clear in the code actually, thanks